pkg/console/starter/starter.go

```diff
@@ -5,6 +5,8 @@ import (
 	"os"
 	"time"
 
+	"github.com/openshift/library-go/pkg/operator/management"
```

Comment:
nit.
lets put this to the rest of the openshift libs

Reply:
+1